### PR TITLE
Retry auto mfa test cases + sleep before making two back to back AUTO MFA requests

### PR DESCRIPTION
**Pull request by shahzaibj**

### CODE REVIEW

File: msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/local/TestCase99656.java [hunk 1]
```diff
@@ -103,6 +106,8 @@ public void handleUserInteraction() {
         silentLatch.await(TokenRequestTimeout.SILENT);
 
         // second interactive request
+        // wait about a minute here to throttle usage of AUTO MFA account
+        Thread.sleep(TimeUnit.MINUTES.toMillis(1));
```
Comment (AdamBJohnsonx) [inline msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/local/TestCase99656.java:110]: ThreadUtils.sleepSafely()?

Comment (shahzaibj) [reply msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/local/TestCase99656.java]: Addressed here: https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1251/commits/fd1a43460545e3ca9127b24b0f85b96cfc8c9022

**Review by AdamBJohnsonx — APPROVED** [review]

File: msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/local/TestCase99656.java [hunk 2]
```diff
@@ -103,6 +108,12 @@ public void handleUserInteraction() {
         silentLatch.await(TokenRequestTimeout.SILENT);
 
         // second interactive request
+        // wait about a minute here to throttle usage of AUTO MFA account
+        ThreadUtils.sleepSafely(
+                (int) TimeUnit.MINUTES.toMillis(1),
+                TAG,
+                "Problem occurred while sleeping safely to throttle AUTO MFA requests."
```
Comment (AdamBJohnsonx) [inline msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/local/TestCase99656.java:115]: I mean, there's nothing really safe about it, it just takes care of the boilerplate for you in terms of thread.sleep.

Comment (shahzaibj) [reply msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/local/TestCase99656.java:115]: Yeah...I couldn't figure out a better message to log here :D

**Pull request merged**